packages/cli-helpers/src/auth/__tests__/authTasks.test.ts

```diff
@@ -0,0 +1,71 @@
+// Have to use `var` here to avoid "Temporal Dead Zone" issues
+// eslint-disable-next-line
+var mockIsTypeScriptProject = true
+global.__dirname = __dirname
+
+jest.mock('../../lib/paths', () => {
+  const path = require('path')
+
+  return {
+    ...jest.requireActual('../../lib/paths'),
+    getPaths: () => {
+      const base = '/mock/base/path'
+
+      return {
+        web: {
+          src: path.join(base, 'web', 'src'),
+        },
+      }
+    },
+  }
+})
+
+jest.mock('../../lib/project', () => ({
+  isTypeScriptProject: () => mockIsTypeScriptProject,
+}))
+
+jest.mock('../../lib', () => ({
+  transformTSToJS: (_path: string, data: string) => data,
+}))
+
+// This will load packages/cli-helpers/__mocks__/fs.js
+jest.mock('fs')
+
+// @ts-expect-error - this is good enough for our mock
+fs.readdirSync = () => {
+  return ['auth.ts.template']
+}
+
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from '../../lib/paths'
+import { createWebAuth } from '../authTasks'
+
+beforeEach(() => {
+  mockIsTypeScriptProject = true
```

Comment:
This should be in the it block no?

Reply:
I put it here to make it TS by default. And also so you wouldn't have to remember setting the variable in every test. If there's a nicer way to do it, please let me know 🙂

Reply:
Hmm understood! The way I read the tests, because there's one test for TS and one test for JS, it would've made sense to me to set it in the TS test.

No issues keeping it this way though!

Reply:
I agree it's a bit weird when there are only two tests. I copied the approach from here, where it might make more sense https://github.com/redwoodjs/redwood/blob/main/packages/cli-helpers/src/auth/__tests__/authFiles.test.ts

Reply:
Those long variable names.... I wonder who wrote that 🤣

Reply:
I have been spending some time in the backend Java code at work. Please don't tell me I'm starting to get infected 🤣